src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -108,14 +116,45 @@ public EmbeddedKafkaBroker start()
         return this;
     }
 
+    // Create consumer offsets ourselves so we can control replication, partitions, etc
+    private void maybeCreateConsumerOffsets() throws InterruptedException {
```

Comment:
Why is this named like it is?  Seems like it's just polling on describe topics.

Reply:
Yeah I think @stevenschlansker pulled this from chat, where there is a condition. The name doesn't make a lot of sense now.